executors/node/collator.js

```diff
@@ -39,13 +44,24 @@ module.exports = {
         result_bool = false;
       }
       outputLine = {'label':json['label'],
-                    'result': result_bool,
-                    'compare_result': compared,
                    }
-
-      if (result != true) {
+      if (result == true) {
```

Comment:
Issue: if `result == false`, it should be returned as a potential test failure. Only if the options are unsupported should an unsupported status be returned.

Reply:
OK. I see that there's an ECMAScript option for sensitivity. I'll see if that fits with any of the "compare_type" options.

And should we set all tests that have explicit 'rules' to be 'unsupported'?

Reply:
You mean like this test?

`{"test_type": "collation_short", "label":"00003","s1":"9","s2":"\u0000","compare_type":"&lt;1","test_description":" simple CEs &amp; expansions","rules":"&amp;\\x01&lt;&lt;&lt;\\u0300&amp;9&lt;\\x00&amp;\\uA00A\\uA00B=\\uA002&amp;\\uA00A\\uA00B\\u00050005=\\uA003","hexhash":"d439f742bdde4b51012ee05519dfde424e25334a"}`

Yeah, explicit "rules" are unsupported.